src/Group.Java

```diff
@@ -0,0 +1,25 @@
+
+public class Group {
+    private String[] students = new String[100];        
+    private int numStudents = 0;  
+
+    public void addStudent(String s) {
```

Comment:
all methods you created should have documentation comments in the form

```java
/** 
 * Method Description
 */
```
The only exception is standard getters and setters, which this is not.